ramalama/config.py

```diff
@@ -5,6 +5,8 @@
 from ramalama.common import container_manager, default_image
 from ramalama.toml_parser import TOMLParser
 
+DEFAULT_PORT = 8080
```

Comment:
We want the default to be 8080-8090

The point is not that we hardcode the tries to 10. Moreso that we hardcode the range to 8080-8090

Reply:
updated with a default range instead, hopefully we are converging to the expected solution